src/app/modules/frontend/frontend.controller.ts

```diff
@@ -13,10 +12,10 @@ const logger = createLoggerWithLabel(module)
  * Handler for GET /frontend/datalayer endpoint.
  * @param req - Express request object
  * @param res - Express response object
- * @returns {String} Templated Javascript code for the frontend to initialise Google Tag Manager
+ * @returns Templated Javascript code for the frontend to initialise Google Tag Manager
  */
 export const addGoogleAnalyticsData: RequestHandler<
-  ParamsDictionary,
+  unknown,
   string | { message: string }
 > = (req, res) => {
```

Comment:
should we add a `unknown` for the body here? otherwise the body is `any` by default

Reply:
same for the other RequestHandlers which don't have a 3rd argument

Reply:
Will create a new type `ControllerHandler` and use that type to replace all `RequestHandler`s, since interface declaration merging cannot override the `any` typings :(

Reply:
Resolved by 3928a2a